src/components/home/SettingsButton.js

```diff
@@ -0,0 +1,29 @@
+import { TouchableOpacity, Dimensions, StyleSheet } from 'react-native'
```

Comment:
same for the Store button, top margin isn't right 

solution: STORE_TOP line becomes `const STORE_TOP = width > 700 ? 110 : 70`, this is to account for the TrainAnimation comp

see 

<img src="https://github.com/Austinae/metrogame/assets/44261381/c2d71001-0ce0-43d0-b68e-dbc1c6e2eecf" width="350" />

Reply:
Like we said in my house, "ça m'a saoulé" so let's see if the solution I've provided suit you 👍